[ADD] [16.0] sale_stock_reservation_issue_on_qty_at_date_widget (#3625)
OCA/sale-workflow: sale_stock_reservation_issue_on_qty_at_date_widget/readme/INSTALL.md

> In order to use this module, you need to install those modules:

Reviewer: ```suggestion
In order to use this module, you need to install the next module:
```
